gtda/time_series/tests/test_embedding.py

```diff
@@ -72,7 +74,27 @@ def test_embedder_transform(parameters_type, expected):
     assert_almost_equal(embedder.fit_transform(signal), expected)
 
 
+def test_embedder_resample():
+    embedder = TakensEmbedding(time_delay=4, dimension=5, stride=3)
+    embedder = embedder.fit(signal)
```

Comment:
No need for "embedder =" here as fit does in-place operations but don't worry

Reply:
I will change it - it's just that I have a slight preference to be explicit (I am not a fan of mutable objects), in case it doesn't change anything.